### PR TITLE
Translate the component page and fixed the error.

### PR DESCRIPTION
**Pull request by warmchang**
1. Translate the component page.
2. Fixed the translation error.

Signed-off-by: William Chang <warmchang@outlook.com>

### CODE REVIEW

File: v2/cn/components.md [hunk 1]
```diff
@@ -8,11 +8,13 @@
...
-下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有的组件可以被激活或禁止。
-大部组件默认是激活的，也有部件不是。
-[LuaJIT](luajit.html)、 [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html) 默认是没有激活的。您需要通过以下选项在编译
-[OpenResty](openresty.html)的时候将它们各自激活， `--with-luajit`、
-`--with-http_drizzle_module`、 `--with-http_postgres_module`和 `--with-http_iconv_module` 。
+下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有组件均可以方便的被激活或禁止。
+
+大部分组件默认是激活的，也有些也不是。
+
+Lua 5.1 版本中, [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html)默认并未开启。 你需要加入`--with-lua51`、`--with-http_drizzle_module`、`--with-http_postgres_module`和`--with-http_iconv_module` 编译选项来开启它们，其余各组件的编译选项，可对照[安装OpenResty](installation.html)说明，按需使能。
```
Comment (agentzh) [inline v2/cn/components.md]: > Lua 5.1 版本中

I'm afraid this is wrong. The ngx_drizzle and ngx_postgres modules have nothing to do with the standard Lua 5.1 interpreter.

Comment (agentzh) [inline v2/cn/components.md]: Your wording is ambiguous and confusing here.

Comment (agentzh) [inline v2/cn/components.md]: Maybe you mean

"标准 Lua 5.1 解释器、"

here instead.

Comment (warmchang) [inline v2/cn/components.md]: Yes, My understanding is "标准 Lua 5.1 解释器、".

Comment (agentzh) [inline v2/cn/components.md]: @warmchang Will you fix that? Your current version looks wrong to me.

**Comment by warmchang** [comment]
@agentzh ,I've done, pls check it, thx.

File: v2/cn/components.md (hunk 1 above)
Comment (warmchang) [inline v2/cn/components.md]: @agentzh ,I've done, pls check it, thx.

**Comment by warmchang** [comment]
Hi, @agentzh ,
Any else should be updated?

File: v2/cn/components.md [hunk 2]
```diff
@@ -8,11 +8,13 @@
...
 --->
 
-下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有的组件可以被激活或禁止。
-大部组件默认是激活的，也有部件不是。
-[LuaJIT](luajit.html)、 [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html) 默认是没有激活的。您需要通过以下选项在编译
-[OpenResty](openresty.html)的时候将它们各自激活， `--with-luajit`、
-`--with-http_drizzle_module`、 `--with-http_postgres_module`和 `--with-http_iconv_module` 。
+下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有组件均可以方便的被激活或禁止。
+
+大部分组件默认是激活的，也有些也不是。
```
Comment (agentzh) [inline v2/cn/components.md]: "也有些也不是" does not read well to me :)

File: v2/cn/components.md [hunk 3]
```diff
@@ -8,11 +8,13 @@
...
+
+标准 Lua 5.1 解释器中, [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html)默认并未开启。 你需要加入`--with-lua51`、`--with-http_drizzle_module`、`--with-http_postgres_module`和`--with-http_iconv_module` 编译选项来开启它们，其余各组件的编译选项，可对照[安装OpenResty](installation.html)说明，按需使能。
+
+在 OpenResty 1.5.8.1 版本之前, 如果 [LuaJIT](luajit.html) 2.x 环境不具备，则将默认采用标准的Lua 5.1解释器。 所以对于早期的老版本，你需要显式的加入`--with-luajit`编译选项 ( 1.5.8.1+版本已默认开启 )来使能 [LuaJIT](luajit.html) 2.x功能.
```
Comment (agentzh) [inline v2/cn/components.md]: This is inaccurate. The older versions of OpenResty always uses the standard Lua 5.1 interpreter by default, regardless of the user environment.

File: v2/cn/components.md [hunk 4]
```diff
@@ -8,11 +8,13 @@
...
-下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有的组件可以被激活或禁止。
-大部组件默认是激活的，也有部件不是。
-[LuaJIT](luajit.html)、 [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html) 默认是没有激活的。您需要通过以下选项在编译
-[OpenResty](openresty.html)的时候将它们各自激活， `--with-luajit`、
-`--with-http_drizzle_module`、 `--with-http_postgres_module`和 `--with-http_iconv_module` 。
+下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有组件均可以方便的被激活或禁止。
+
+大部分组件默认是激活的，也有些也不是。
+
+标准 Lua 5.1 解释器中, [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html)默认并未开启。 你需要加入`--with-lua51`、`--with-http_drizzle_module`、`--with-http_postgres_module`和`--with-http_iconv_module` 编译选项来开启它们，其余各组件的编译选项，可对照[安装OpenResty](installation.html)说明，按需使能。
```
Comment (agentzh) [inline v2/cn/components.md]: You did not translate the "respectively" word in the original English text, I'm afraid :)

**Comment by warmchang** [comment]
@agentzh , I revised the mistakes by your opinion, pls check it, thanks.

BTW, in the English version, what is the mean of "while LuaJIT 2.x is not.".

> Before the 1.5.8.1 release, the standard Lua 5.1 interpreter is enabled by default while LuaJIT 2.x is not.

**Comment by agentzh** [comment]
@warmchang "while LuaJIT 2.x is not" means "while LuaJIT 2.x is not enabled by default".

File: v2/cn/components.md [hunk 5]
```diff
@@ -8,11 +8,13 @@
...
-下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有的组件可以被激活或禁止。
-大部组件默认是激活的，也有部件不是。
-[LuaJIT](luajit.html)、 [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html) 默认是没有激活的。您需要通过以下选项在编译
-[OpenResty](openresty.html)的时候将它们各自激活， `--with-luajit`、
-`--with-http_drizzle_module`、 `--with-http_postgres_module`和 `--with-http_iconv_module` 。
+下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有组件均可以方便的被激活或禁止。
+
+绝大部分组件已内置在OpenResty安装包中，但也有一部分不包含在内。
+
+标准 Lua 5.1 解释器中, [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html)默认并未开启。 你需要分别加入`--with-lua51`、`--with-http_drizzle_module`、`--with-http_postgres_module`和`--with-http_iconv_module` 编译选项来开启它们，其余各组件编译选项，可对照[安装OpenResty](installation.html)说明，按需使能。
```
Comment (agentzh) [inline v2/cn/components.md]: "使能“? Alas.

File: v2/cn/components.md [hunk 6]
```diff
@@ -8,11 +8,13 @@
...
+
+在 1.5.8.1 版本之前, OpenResty将默认使用标准的Lua 5.1解释器。 所以对于老版本，你需要显式的加入`--with-luajit`编译选项 ( 1.5.8.1+版本已默认开启 )来使能 [LuaJIT](luajit.html) 2.x功能.
```
Comment (agentzh) [inline v2/cn/components.md]: "将" looks wrong to me. We're talking about the old history here, right?

Comment (warmchang) [inline v2/cn/components.md]: ok, i will thange it to "会".

File: v2/cn/components.md (hunk 5 above)
Comment (warmchang) [inline v2/cn/components.md]: In fact, in many scenarios, the function enable / disable will be called 使能 / 去使能.
But any way, there are not mentioned about the other build config in English version, so how do you think to translate the sentence like the followed:

> 你需要在[安装OpenResty](installation.html)时，分别加入`--with-lua51`、`--with-http_drizzle_module`、`--with-http_postgres_module`和`--with-http_iconv_module` 编译选项，以开启对应的特性。

Comment (agentzh) [inline v2/cn/components.md]: "使能" is a Taiwanese term? I've never seen this word in my life though :)

**Comment by agentzh** [comment]
Since there've been too many back and forth. I've tweaked your translation myself and committed to master. Thanks for your contribution!

**Comment by warmchang** [comment]
ok, thank you!

**Pull request closed without merging**